spaceship/lib/spaceship.rb

```diff
@@ -2,6 +2,7 @@
 require 'spaceship/base'
 require 'spaceship/client'
 require 'spaceship/launcher'
+require 'fastlane_core'
```

Comment:
Oh That was introduced to use the FastlaneCore::Helper for messages.   What should be used instead of the Helper?